dataactbroker/handlers/accountHandler.py

```diff
@@ -717,4 +720,33 @@ def setSkipGuide(self, session):
             exc = ResponseException("skip_guide must be a boolean", StatusCode.CLIENT_ERROR)
             return JsonResponse.error(exc, exc.status)
         userDb.session.commit()
-        return JsonResponse.create(StatusCode.OK,{"message":"skip_guide set successfully","skip_guide":skipGuide})
\ No newline at end of file
+        return JsonResponse.create(StatusCode.OK,{"message":"skip_guide set successfully","skip_guide":skipGuide})
+
+    def emailUsers(self, system_email, session):
+        """ Send email notification to list of users """
+        requestDict = RequestDictionary(self.request)
+        if not (requestDict.exists("users") and requestDict.exists("submission_id")):
+            exc = ResponseException("Email users route requires 'users' and 'submission_id'", StatusCode.CLIENT_ERROR)
+            return JsonResponse.error(exc, exc.status)
+
+        uid = session["name"]
+        current_user = self.interfaces.userDb.getUserByUID(uid)
+
+        user_ids = requestDict.getValue("users")
+        submission_id = requestDict.getValue("submission_id")
+        self.jobManager.getSubmissionById(submission_id)
```

Comment:
Good idea to check for a submission id here. If the submission is invalid, does the job handler send the appropriate message back to the front end, and is the front end expecting such a message? (it's hard to imagine the front end sending a bad id to begin with, but want to make sure everything is hooked up)